hwilib/devices/ledger.py

```diff
@@ -1,26 +1,33 @@
 # Ledger interaction script
 
-from .hwwclient import HardwareWalletClient
+from ..hwwclient import HardwareWalletClient
 from btchip.btchip import *
 from btchip.btchipUtils import *
 import base64
 import json
 import struct
-from . import base58
-from .serializations import hash256, hash160, ser_uint256, PSBT, CTransaction, HexToBase64
+from .. import base58
+from ..base58 import get_xpub_fingerprint_hex
+from ..serializations import hash256, hash160, ser_uint256, PSBT, CTransaction, HexToBase64
 import binascii
 import logging
 
+LEDGER_VENDOR_ID = 0x2c97
+LEDGER_DEVICE_ID = 0x0001
+
 # This class extends the HardwareWalletClient for Ledger Nano S specific things
 class LedgerClient(HardwareWalletClient):
 
     # device is an HID device that has already been opened.
```

Comment:
you can delete these stale lines, especially now the initialization is done locally.

Reply:
Removed